tests/acceptance/course-page/code-examples/export-to-github-test.js

```diff
@@ -0,0 +1,173 @@
+import catalogPage from 'codecrafters-frontend/tests/pages/catalog-page';
+import codeExamplesPage from 'codecrafters-frontend/tests/pages/course/code-examples-page';
+import coursePage from 'codecrafters-frontend/tests/pages/course-page';
+import createCommunityCourseStageSolution from 'codecrafters-frontend/mirage/utils/create-community-course-stage-solution';
+import percySnapshot from '@percy/ember';
+import testScenario from 'codecrafters-frontend/mirage/scenarios/test';
+import { setupAnimationTest } from 'ember-animated/test-support';
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'codecrafters-frontend/tests/helpers';
+import { signIn } from 'codecrafters-frontend/tests/support/authentication-helpers';
+import { settled } from '@ember/test-helpers';
+import courseOverviewPage from 'codecrafters-frontend/tests/pages/course-overview-page';
+
+module('Acceptance | course-page | code-examples | export-to-github', function (hooks) {
+  setupApplicationTest(hooks);
+  setupAnimationTest(hooks);
+
+  async function setupTestScenario(server) {
+    testScenario(server);
+    const currentUser = signIn(this.owner, server);
+
+    const solutionAuthor = server.create('user', {
+      avatarUrl: 'https://github.com/author.png',
+      createdAt: new Date(),
+      githubUsername: 'author',
+      username: 'author',
+    });
+
+    const redis = server.schema.courses.findBy({ slug: 'redis' });
+    const python = server.schema.languages.findBy({ slug: 'python' });
+
+    server.create('repository', 'withFirstStageCompleted', {
+      course: redis,
+      language: python,
+      user: currentUser,
+    });
+
+    return {
```

Comment:
This returns 5 different values, and only one of them actually gets used - could just have been `return createCommunityCourse....`